db/migrate/20180426143346_add_first_and_last_names.rb

```diff
@@ -0,0 +1,6 @@
+class AddFirstAndLastNames < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :first_name, :string, null: false
+    add_column :users, :last_name, :string, null: false
+  end
+end
```

Comment:
In fact, we really shouldnt be adding any columns to users, since I dont see an existing users migration, but there is an existing. Looks like your config is a bit off.

Reply:
Combined with username migration